requirements.txt

```diff
@@ -35,6 +35,9 @@ google-api-python-client==1.6.4
 Babel==2.5.1
 citeproc-py==0.4.0
 boto3==1.4.7
+#Using own fork of django-waffle until https://github.com/jsocol/django-waffle/pull/259 is merged
```

Comment:
Looks like that PR has been replaced by https://github.com/jsocol/django-waffle/pull/266 .

No need to change your fork for now, though--it's a simple change, and hopefully the fix will be released upstream soon.